draft-ietf-cbor-7049bis.md

```diff
@@ -514,30 +538,30 @@ referred to as "streaming" within a data item.)
 Indefinite-length arrays and maps are dealt with differently than
 indefinite-length byte strings and text strings.
 
+### The "break" stop code {#break}
+
+The "break" stop code is encoded with major type 7 and additional
+information value 31 (0b111_11111). It is not itself a data item: it
+is just a syntactic feature to close an indefinite-length item.
+
+If the "break" stop code appears where an item is expected, for
+example directly inside a definite-length array or map, the enclosing
```

Comment:
I think this sentence needs some clarification.  In an indefinite array, I am expecting an item but can also take a break.  This seems to be covered by this sentence as a subset.  Perhaps 'where only an item is permitted,'

Reply:
Good point. I'm worried that "where only an item is permitted" winds up being circular. How about "anywhere other than directly inside an indefinite-length string, array, or map"? I could also be a little less precise with "in a context that does not explicitly handle it" if we think more contexts are likely to allow it in the future.